src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="#">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
 
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a href="#" class="nav__link">Apple</a>
```

Comment:
if you wanted to improve the link, it is better to add a unique anchor to each link
```suggestion
            <a href="#apple" class="nav__link">Apple</a>
```

Reply:
> Almost done) You did smart solution. Let's improve Fix media queries ![image](https://user-images.githubusercontent.com/70009205/227251038-19bf0ad0-b0a9-4fec-815b-f344976ddf66.png)

Thanks for you review!) i corrected the code=)